project-base/tests/App/Acceptance/acceptance/PageObject/Front/ProductListPage.php

```diff
@@ -41,6 +41,9 @@ public function addProductToCartByName($productName, $quantity = 1)
         $context = $this->getProductListCompomentContext();
 
         $this->productListComponent->addProductToCartByName($productName, $quantity, $context);
+
+        $this->tester->waitForAjax();
+        $this->tester->wait(1);
```

Comment:
is this a common rule to wait one extra second after ajax? 
I'm thinking about moving this extra wait into waitForAjax method.

Reply:
Well we might think of adding that to waitForAjax, I would not mind to have higher wait times, cause it could fix many problems. But in this case it is waitingForAjax and it gets data for "Product added to cart" window, but in that time that window might not be fully opened, so that is that extra second for that.